docs/plots/index.rst

```diff
@@ -308,6 +308,15 @@ line position (which corresponds to `mdl.pars[2]`):
     WARNING: hard maximum hit for parameter line.ampl
     >>> iproj.plot()
 
+Customizing plotting further
+============================
```

Comment:
```suggestion
Customizing plots
=================
```

Reply:
I understand the markup here, but I don't like the suggestion either. There is a stuff about customizing plots above that (e.g .the section "Changing the plot appearance"), so I wanted to make a section heading that implies "customizing more than explained above". 

At this time, I think "plotting with different backends" might be OK (since the term backend is explained in that section and is not that uncommon in computer use). When I wrote this, I has a structure in mind like [this](https://docs.astropy.org/en/stable/io/ascii/index.html), where you have a general introduction and then subpages. Here "backends" is the only subpage right now, but there could be more in the future ("plotting datastacks", the "A sample of plots"-notebook could go here). But maybe I'm overthinking it and I should jsut say "backends" for now and change it if we add more subpages.

Reply:
I'll change this to "More plot customization" for now, but I'm happy to take more suggestions in the next round of review.